code/modules/atmospherics/machinery/components/fusion/hfr_procs.dm

```diff
@@ -464,3 +554,48 @@
 	if(!origin_turf)
 		return
 	origin_turf.assume_air(remove_mixture)
+
+/obj/machinery/atmospherics/components/unary/hypertorus/core/proc/check_spill(delta_time)
+	var/obj/machinery/atmospherics/components/unary/hypertorus/cracked_part = check_cracked_parts()
+	if (!cracked_part)
```

Comment:
Why not handle cracked behavior inside the if block, since there's less lines of code there, and it makes more logical sense

Reply:
Will swap that around shortly

Reply:
Done